src/components/ImageView/index.js

```diff
@@ -22,7 +22,7 @@ class ImageView extends PureComponent {
         this.state = {
             containerHeight: 0,
             containerWidth: 0,
-            isZoomed: false,
+            isZoomed: true,
```

Comment:
@ahmdshrif can we go with an alternative approach because it feels like a workaround, and it's buggy. Let's keep isZoomed be `false` by default.

It's still buggy for normal images. What do you think of using imgTop, imgLeft, etc instead.

https://user-images.githubusercontent.com/29673073/153427218-efb04211-fd94-4c3b-a55e-b85439deb611.mp4

Reply:
it must not get zooming if it's small.  Let me try to test with different image sizes. on my side. 
I think imgTop, imgLeft,  is more related to position when you move the image. but let me check this.

Reply:
@rushatgabhane  can you share with me the image that have the issue because for me I try many image sizes and still work fine with me

https://user-images.githubusercontent.com/21364901/153516369-ce66c729-f945-4520-9eb5-398a6f45e51e.mov

Reply:
Here you go ![RDT_20220131_1353072297764153181649913](https://user-images.githubusercontent.com/29673073/153517495-5532dfa3-5e41-4714-876e-b7613d05b893.jpg)

Reply:
Cute puppy 😍

Reply:
@rushatgabhane  ok thanks, I fix t it  now

https://user-images.githubusercontent.com/21364901/153518142-ebf34fbe-f466-45e6-879e-b0e00f318b51.mov

Reply:
> Cute puppy 😍

ikrr 😍





> @rushatgabhane ok thanks, I fix t it now

Fantastic!